### ISSUE TITLE
Returning frequentist MMRM model object

### ISSUE THREAD

**Issue by gowerc** [insightsengineering/rbmi#274]
Hi @wolbersm ,

We have been asked a few times by Paul to add return the model object of the MMRM fit for the bootstrap methods  for diagnostic / residual purposes.   

Talking to @nociale our current proposal would be that we return the MMRM fit to the original dataset (only) for the condmean jackknife + bootstrap methods,   for all other methods  (bayesian, approx bayes, blmi) we would carry on returning nothing.

What do you think ?

**Comment by wolbersm** [comment]
We need to fit an MMRM model to the original data for all methods, right? 
Therefore, I'd suggest to return the original MMRM fit for all methods. 
Does this work or would it cause extra complications.

One totally separate issue is that the MMRM to the original data does exclude "non-MAR" post-ICE data but I guess this is unavoidable. Also, it's not 100% clear to me how to do good residual diagnostics for an MMRM model but I leave that to Paul for now.

**Comment by wolbersm** [comment]
Thanks for clarifying @gowerc 
I am not 100% sure what Paul wants to do with the MMRM fit in any case but after some consideration, your original proposal sounds reasonable.

**Comment by nociale** [comment]
@gowerc @wolbersm Just to be clear, currently `drawsObj$fit` is `NULL` for all the methods except `bayes` for which the `stanfit` object is returned. We could return the fit object from the `glmmTMB()` function for the MMRM fit on the original dataset for the `condmean` method (either bootstrap or jaccknife). `drawsObj$fit` would still be `NULL` for `bmlmi` and `approxbayes`. Even though we do fit MMRM on the original dataset also when these methods are called, in these cases (as for `bayes`) this fit serves only for initialization / setting starting values, so I would personally not output this fit object.

**Comment by wolbersm** [comment]
Thanks, as long as the content of `drawsObj$fit` is properly documented, this sounds ok for me.

**Comment by gowerc** [comment]
I'm not sure we would be able to document it other than saying its a "glmfit" object. Otherwise we have a moving target that our documentation is dependent on another teams implementation details.

**Comment by wolbersm** [comment]
Yes, I think it's fine to just say that this contains the result of the imputation model fit returned by glmmTMB() or stan().

**Comment by delmarp** [comment]
sounds good. I would also return the full object from the ancova function for the same reason to check residuals

**Comment by pengguanya** [comment]
Just add here the relevant functions in case I forgot. There is a kind of chain relation among them
```
1. draws.condmean
2. get_draws_mle
3. get_mmrm_sample
4. fit_mmrm_multiopt
5. fit_mmrm
6. glmmTMB
```
